docs/pages/en/4.providers/imagekit.md

```diff
@@ -18,3 +18,271 @@ export default {
   }
 }
 ```
+
+## ImageKit `fit` Parameters
+
+In addition to the standard [`fit`](/components/nuxt-img#fit) properties of Nuxt Image and Nuxt Picture, ImageKit offers more cropping and resizing options to the users:
+
+* `extract` - The output image has its height, width as requested and the aspect ratio is preserved. Unlike `cover` parameter, here we extract out a region of the requested dimension from the original image.
+* `pad_extract` - This parameter is similar to `extract`. This comes in handy in scenarios where we want to extract an image of larger dimension from a smaller image. So, the `pad_extract` mode adds a solid colored padding around the image to make it match the exact size requested.
```

Comment:
See if you can improve this definition. The bit about this type of extraction not being practically possible without padding could be included.

Reply:
I think that bit is obvious from the statement. (also this- 
> Our expectation would be that user is familiar with at least the basics of ImageKit transformations before trying to integrate it into Nuxt).

I feel getting into more details of its working is under the scope of IK docs, and just below it there is a link to our crop documentation guide.